tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+
+1. Create a file with the contents below and name it `rbac-setup.yaml`. The following example applies to EKS clusters.
+
+```console
+apiVersion: eksctl.10/vialphas
+kind: ClusterConfig
+metadata:
+    name: "CLUSTER-NAME"
+    region: "AWS-REGION"
+identitvProviders:
+  - name: autho
+    type: oidc
+    issuerUrl: "ISSUER-URL"
+    clientId: "CLIENT-ID"
+    usernameClaim: email
+```
+Where:
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `AWS-REGION` is the AWS region of the EKS cluster
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+
+
+2. Using `eksctl`, execute the following command:
+
+```console
+eksctl associate identityprovider -f rbac-setup.yaml
+```
+
+3. To check if the association of the OIDC provider with the EKS cluster was successful, run the following command
+
+```console
+eksctl get identityprovider --cluster `CLUSTER-NAME`
+```
+
+The output should state `ACTIVE` in the STATUS column.
+
+
+## <a id="configure-tap-gui"></a> Configure the Tanzu Application Platform GUI
+
+The next step is to configure visibility of the remote cluster in Tanzu Application Platform GUI. 
+
+1. Make sure you added an `auth` section to the `app_config` file taht Tanzu Application Platform GUI uses. In the example for Auth0, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+auth:
+    environment: development
+    providers:
+        autho:
+            development:
+                clientId: "CLIENT-ID"
+                clientSecret: "CLIENT-SECRET"
+                domain: "ISSUER-URL"
+```
+Where:
+
+   - `CLIENT-ID` is the Client ID you obtained while setting up the OIDC provider
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
+   - `CLIENT-SECRET` is the Client Secret you obtained while setting up the OIDC provider
+
+
+2. You must also add a `kubernetes` section to the `app_config` file that Tanzu Application Platform GUI uses. This section must have an entry for each cluster that has resources to view.
+
+To do so, copy this YAML content into `tap-gui-values.yml`:
+
+
+```console
+kubernetes:
+    serviceLocatorMethod:
+        type: 'multiTenant'
+    clusterLocatorMethods:
+      - type: 'config'
+        clusters:
+          - name: "CLUSTER-NAME"
+            url: "CLUSTER_URL"
+            authProvider: oidc
+            oidcTokenProvider: auth0
+            skipTLSVerify: true
+            skipMetricsLookup: true
+```
+Where:
+
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `CLUSTER_URL` is the URL for the remote cluster you are connecting to Tanzu Application Platform GUI. To obtain your cluster's URL, execute the following command:
+
+```console
+CLUSTER_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+
+echo CLUSTER_URL: $CLUSTER_URL
+```
```

Comment:
fix placeholder name
~~~suggestion
   - `CLUSTER-URL` is the URL for the remote cluster you are connecting to Tanzu Application Platform GUI. To obtain your cluster's URL, execute the following command:

```console
CLUSTER_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')

echo CLUSTER-URL: $CLUSTER_URL
```
~~~

It may also be worth noting that this snippet gets you the URL of the first configured cluster in your kubeconfig. For users with multiple clusters in their kubeconfig -- especially when their current context is not targeting the _first_ of those clusters -- might encounter unexpected results here.

Reply:
I'm guessing this part was copy-pasted from [this snippet](https://github.com/pivotal/docs-tap/blob/364812c0c9b7af6c27c8bb3cb32cc34f268042d4/tap-gui/cluster-view-setup.md?plain=1#L135), so maybe we can deep link to that section instead of duplicating here? Then we can focus this doc purely on the `authProvider` part of the config.

Reply:
Thanks, I've re-ordered this procedure so that this is at the top. I can also see there's more detail now. 
See https://docs-staging.vmware.com/en/draft/Tanzu-Application-Platform/1.2/tap/GUID-tap-gui-tap-gui-rbac-setup-tap-gui-rbac-eks.html#configure-tap-gui if you're interested, and if you'd like to make changes.